### PR TITLE
more simplifications to bokeh module

### LINKED ISSUE #2051: simplify bokeh/__init__.py even more

### PR DESCRIPTION
**Pull request by bryevdv**
issues: closes #2051

### CODE REVIEW

**Comment by bryevdv** [comment]
@bokeh/core this is ready for review @damianavila interested in your input in particular, I think given that `report_issue` unfortunately has not met any adoption, that removing it makes sense so there is less to document and maintain.

File: sphinx/source/conf.py [hunk 1]
```diff
@@ -64,16 +64,8 @@
 # built documents.
 #
 # Let's try to automatically get the version
```
Comment (fpliger) [inline sphinx/source/conf.py]: should this comment be removed?

File: bokeh/util/tests/test_testing.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
```
Comment (fpliger) [inline bokeh/util/tests/test_testing.py]: jeje.... funny name :smile:

**Comment by fpliger** [comment]
very minor comment. LGTM

**Comment by bryevdv** [comment]
@damianavila I will merge this Monday afternoon CST unless there are comments/objections.

File: bokeh/util/tests/test_testing.py (hunk 2 above)
Comment (bryevdv) [inline bokeh/util/tests/test_testing.py]: Hah, yah I know but I am trying to be consistent with the naming of test modules :)

File: sphinx/source/conf.py [hunk 3]
```diff
@@ -64,16 +64,8 @@
...
-    __version__ = conda_version.replace("'","")
-    del conda_version
-except ImportError:
-    __version__ = get_versions()['version']
-    del get_versions
+from bokeh import __version__
+from bokeh.settings import settings
 
 # if you need to redeploy the released docs, you only need the x.x.x version
 if settings.released_docs():
```
Comment (bryevdv) [inline sphinx/source/conf.py]: If this is true it does:
```
 __version__ = __version__.split('-')[0]
``` 
Is this correct? Shouldn't it just unconditionally use the value of `settings.released_docs()` as the version?

File: bokeh/util/testing.py [hunk 4]
```diff
@@ -89,115 +86,3 @@ def runtests(verbosity=1, xunitfile=None, exit=False):
 
     # Run the tests
     return nose.main(argv=argv, exit=exit)
-
-
-def report_issue(number=None, owner="bokeh", repo="bokeh",
```
Comment (damianavila) [inline bokeh/util/testing.py]: ok... I agree, nobody seems to be using it... so let's get rid of it...

File: sphinx/source/conf.py [hunk 5]
```diff
@@ -64,16 +64,8 @@
...
-from bokeh import settings
-
-try:
-    from bokeh.__conda_version__ import conda_version
-    __version__ = conda_version.replace("'","")
-    del conda_version
-except ImportError:
-    __version__ = get_versions()['version']
-    del get_versions
+from bokeh import __version__
```
Comment (damianavila) [inline sphinx/source/conf.py]: Why you are deleting this from `conf.py`, it automatically get the correct version... it is important from the devel docs point of view...

File: sphinx/source/conf.py (hunk 1 above)
Comment (damianavila) [inline sphinx/source/conf.py]: yep... I think so...

File: sphinx/source/conf.py (hunk 5 above)
Comment (bryevdv) [inline sphinx/source/conf.py]: handled in one place in `bokeh.util.version` now

Comment (damianavila) [inline sphinx/source/conf.py]: OK, you moved this one to utils...

**Comment by damianavila** [comment]
LGTM

**Comment by bryevdv** [comment]
Great will merge after tests pass.

**Pull request merged**